operations/patch.go

```diff
@@ -97,7 +97,7 @@ func Patch() cli.Command {
 			setPlainLogger,
```

Comment:
I meant to update [this CLI help text](https://github.com/evergreen-ci/evergreen/blob/42006001439d4e32067295869a3b619859f71966/operations/flags.go#L282-L285). You can see the confusing text in `evergreen patch --help`